.dev/reverse-imports-no-repos

```diff
@@ -1,14 +1,15 @@
-adaptalint
-admiral
-autoharp
-aws.alexa
-biolink
-ConNEcT
-DataFakeR
-datarobot
-DominoDataCapture
-NHSRplotthedots
-Plasmidprofiler
-SamplerCompare
-TDA
-wavefunction
+package,maintainer
+adaptalint,conway.max1@gmail.com
```

Comment:
Found it:
https://github.com/maxconway/adaptalint

Reply:
nice!

suggest this as the approach for cases like this:

https://github.com/maxconway/adaptalint/pull/2

Reply:
definitely highlights some manual friction that's bound to happen as the boundary between the packages we can check automatically and those we track manually changes over time

Reply:
Added more PRs:

 - https://github.com/cloudyr/aws.alexa/pull/18
 - https://github.com/compTAG/r-tda/pull/41
 - https://github.com/openpharma/DataFakeR/pull/3
 - https://github.com/phac-nml/plasmidprofiler/pull/1
 
Two more are already added in the dev versions:

 - https://github.com/nhs-r-community/NHSRplotthedots/blob/e54f3d6ab9ce497d2a625e34ab88dae3481ee5c5/DESCRIPTION#L20
 - https://github.com/pharmaverse/admiral/blob/07189fcbbd13a585456fdd826c5d3a7bfe92fe68/DESCRIPTION#L48

Since there's so few left I'll just e-mail the maintainers of the rest:

 - autoharp
 - biolink
 - ConNEcT
 - datarobot
 - DominoDataCapture
 - SamplerCompare
 - wavefunction

Reply:
`SamplerCompare`/`wavefunction`, `autoharp`, and `datarobot` maintainers all wrote back confirming no public repo, so I think it's safe to just use the CRAN mirror as a back-up for those and just report deltas.